src/components/typography/typography.component.tsx

```diff
@@ -11,6 +11,7 @@ const VARIANT_TYPES = [
   "h3",
   "h4",
   "h5",
+  "h6",
```

Comment:
this isn't a suggestion so much as a thought: I recognise we have to add this here to allow `h6` to be used for `Heading` since that uses Typography internally. But was there a reason we didn't allow `h6` for `Typography` before? It's hard to imagine there was a particularly good one, but we should try to find out and look for an alternative if we don't want to expose this on `Typography`.